scripts/install-chef-suse.sh

```diff
@@ -390,6 +390,8 @@ if [ -n "$IPv4_addr" ]; then
     else
         NETWORK_JSON=/opt/dell/chef/data_bags/crowbar/template-network.json
     fi
+    # allow using old json files # TODO: drop in 2017
+    sed -i -e 's/bc-template-network/template-network/' $NETWORK_JSON
```

Comment:
can we add a bit more context to the sed so that it doesn't accidentally match anything else?